packages/ui/components/Pages/Index/MainSection.tsx

```diff
@@ -0,0 +1,10 @@
+import { ChildrenProps } from 'types';
```

Comment:
Let's call the parent folder as `Pages/Program/MainSection.tsx` -> This is actually not the index page, but instead the Program page - Refer to #8 for terminologies

Reply:
@sanathkr `pages/index.tsx` refers to `/` route, so that would mean the current dashboard be `/program` route?

Reply:
Yes, that's correct